### PR TITLE
upgrade angular to angular 13

### PR DESCRIPTION
**Pull request by mhenkens**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ x Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Upgrade to angular13 no functional changed

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

work with angular 13

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

The application using stark should run on angular13.

### CODE REVIEW

File: packages/stark-build/config/webpack.config.js [hunk 1]
```diff
@@ -172,7 +172,9 @@ module.exports = (config, options) => {
 			? {
 					devServer: {
 						// See: https://webpack.js.org/configuration/dev-server/#devserverwritetodisk-
-						writeToDisk: true,
+						devMiddleware: {
```
Comment (SuperITMan) [inline packages/stark-build/config/webpack.config.js:175]: Could you adapt the documentation URL ?

```suggestion
// See: https://webpack.js.org/configuration/dev-server/#devserverdevmiddleware
devMiddleware: {
	writeToDisk: true
},
```

File: showcase/package.json [hunk 2]
```diff
@@ -132,16 +132,16 @@
...
-    "@nationalbankbelgium/eslint-config": "^12.0.0-rc.4",
-    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.2.0-d47412d2d-1703255519.tgz",
-    "@nationalbankbelgium/stark-testing": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.2.0-d47412d2d-1703255519.tgz",
+    "@nationalbankbelgium/eslint-config": "^13.0.0-rc.0",
+    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.2.0-ef22c6222-1704360706.tgz",
+    "@nationalbankbelgium/stark-testing": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.2.0-ef22c6222-1704360706.tgz",
     "@types/core-js": "~2.5.4",
     "@types/hammerjs": "~2.0.39",
-    "@types/node": "^12.20.24",
+	"@types/node": "^18.19.4",
```
Comment (SuperITMan) [inline showcase/package.json:144]: Could you reformat this file ?

File: showcase/angular.json [hunk 3]
```diff
@@ -62,8 +62,12 @@
                 "input": "src/styles/styles.scss"
               }
             ],
+            "stylePreprocessorOptions": {
+              "includePaths": [
+                "node_modules"
+              ]
+            },
```
Comment (SuperITMan) [inline showcase/angular.json:69]: Should this change be part of the starter/angular.json too ?

**Review by SuperITMan — CHANGES_REQUESTED** [review]
Small changes to apply.

File: starter/src/app/app.component.scss [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@use "~@nationalbankbelgium/stark-ui/styles/media-queries" as *;
+@use "@nationalbankbelgium/stark-ui/styles/media-queries" as *;
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply starter/src/app/app.component.scss:1]: done

File: starter/src/styles/_theme.scss [hunk 5]
```diff
@@ -6,7 +6,7 @@ Import the local variables file first to set the correct variables, see:
 @use "variables";
 @use "sass:map";
 
-@use "~@angular/material" as mat;
+@use "@angular/material" as mat;
 @use "~@nationalbankbelgium/stark-ui" as stark-ui;
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply starter/src/styles/_theme.scss:10]: done

File: packages/stark-build/config/webpack.config.js (hunk 1 above)
Comment (mhenkens) [reply packages/stark-build/config/webpack.config.js:175]: done

File: showcase/package.json (hunk 2 above)
Comment (mhenkens) [reply showcase/package.json:144]: done

File: package.json [hunk 6]
```diff
@@ -236,5 +236,8 @@
       "path": "node_modules/cz-customizable"
     },
     "nightlyVersion": "11.0.0"
+  },
+  "dependencies": {
+    "parse5": "^7.1.2"
```
[the remark this answers is not in the document]
Comment (mhenkens) [reply package.json:241]: was needed to perform the migration but i can remove it

**Review by SuperITMan — APPROVED** [review]

**Pull request merged**